services/galley/src/Galley/API/Query.hs

```diff
@@ -314,12 +319,24 @@ getConversationRoles lusr cnv = do
   pure $ Public.ConversationRolesList wireConvRoles
 
 conversationIdsPageFromUnqualified ::
-  Member (ListItems LegacyPaging ConvId) r =>
+  Members
+    [ ListItems LegacyPaging ConvId,
+      ConversationStore,
+      MemberStore,
+      TeamStore
+    ]
+    r =>
   Local UserId ->
   Maybe ConvId ->
   Maybe (Range 1 1000 Int32) ->
   Sem r (Public.ConversationList ConvId)
 conversationIdsPageFromUnqualified lusr start msize = do
+  void $
+    E.getUserTeams (tUnqualified lusr) >>= \tids ->
+      runError @InternalError $
```

Comment:
Can the `InternalError` member from `getGlobalTeamConversation` not be dropped? Looks like it's just doing DB access